<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Composer</class>
 <widget class="QWidget" name="Composer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>631</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>89</width>
    <height>133</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="ComposerGraph">
     <item>
      <layout class="QVBoxLayout" name="SignalComponents">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="ComposingSignal">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="ComposerControl">
     <item>
      <widget class="QGroupBox" name="Components">
       <property name="title">
        <string>Components</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="DeleteButton">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(164, 0, 0);</string>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ComponentsList">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Amplitude">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Amplitude</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="AmplitudeSpinBox">
            <property name="maximum">
             <double>999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Phase">
          <property name="spacing">
           <number>6</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="PhaseLabel">
            <property name="minimumSize">
             <size>
              <width>14</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>102</width>
              <height>26</height>
             </size>
            </property>
            <property name="text">
             <string>Phase</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QDoubleSpinBox" name="PhaseSpinBox">
            <property name="maximum">
             <double>999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Frequency">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Frequency</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="FreqSpinBox">
            <property name="maximum">
             <double>999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="AddWaveBtn">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>209</width>
         <height>76</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>176</height>
        </size>
       </property>
       <property name="title">
        <string>Examples</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="SavedExList">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SaveExampleButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Save Example</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="StartSamplingButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Start Sampling</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
